config/schemas/6/rapidast_schema.json

```diff
@@ -0,0 +1,605 @@
+                                                    "username",
+                                                    "password",
+                                                    "loginPageUrl",
+                                                    "verifyUrl"
+                                                ]
+                                            }
+                                        }
+                                    }
+                                }
+                            ]
+                        },
+                        "container": {
+                            "oneOf": [
+                                {
+                                    "type": "object",
+                                    "properties": {
+                                        "type": {
+                                            "type": "string",
+                                            "default": "none",
+                                            "enum": [
+                                                "podman",
+                                                "none"
+                                            ]
+                                        },
+                                        "parameters": {
+                                            "type": "object",
+                                            "properties": {
+                                                "validReturns": {
+                                                    "type": "array",
+                                                    "items": {
+                                                        "type": "integer"
+                                                    }
+                                                },
+                                                "image": {
+                                                    "type": "string"
+                                                },
+                                                "command": {
+                                                    "type": "string"
+                                                },
+                                                "podName": {
+                                                    "type": "string"
+                                                },
+                                                "volumes": {
+                                                    "type": "array",
+                                                    "items": {
+                                                        "type": "integer"
+                                                    }
+                                                }
+                                            },
+                                            "required": []
+                                        }
+                                    }
+                                },
+                                {
```

Comment:
According to the [config-template-multi-scan](https://github.com/RedHatProductSecurity/rapidast/blob/validate-config-jsonschema/config/config-template-multi-scan.yaml#L30) config file, the container block can be empty, which would not validate against the `object` type, so adding type `null`

Reply:
This 'container' option is highly likely to be changed in the short term (with the podman mode retired). Will we need to consider it now?

Reply:
agree: `I think the schema should match the current version of the config mode`